packages/material-ui/src/Slider/Slider.js

```diff
@@ -247,11 +247,12 @@ export const styles = theme => ({
     '&::after': {
       position: 'absolute',
       content: '""',
-      // reach 48px touch target (2 * 18 + thumb circumference)
-      left: -18,
-      top: -18,
-      right: -18,
-      bottom: -18,
+      borderRadius: '50%',
+      // x10 thumb surface (from 12px^2 to 40px^2).
```

Comment:
```suggestion
      // reach 40px hit target (2 * 14 + thumb diameter)
```

circumference was wrong but the current comment doesn't make much sense either (`^` is for 
potentiation).